backend/README.md

```diff
@@ -30,30 +28,22 @@ npm i
 ```
 
 
+
 ## Run
 
 In order to run the app, you will need a Postgresql database to connect to. The easiest way to get started is to use Tilt (see instructions below), but you can create a local dockerized DB manually by running:
 
 ```sh
-docker-compose up -d postgres
+docker run -d -p 5432:5432 -e POSTGRES_DB=housinghub -e POSTGRES_USER=app -e POSTGRES_PASSWORD=apppassword postgres:11.7
 ```
 
 Now you can run the Flask app by running 
 
 ```sh
-export FLASK_APP=src/server.py
-export FLASK_ENV=development
-export TOKEN_EXP_SECONDS=1000
-export DB_IN_MEMORY_ONLY=false
-export PORT=5000
-export DB_HOST="localhost"
-export DB_PORT=5432
-export DB_USER="app"
-export DB_PASSWORD="apppassword"
-export APP_ENV="local"
+export APP_ENV=local
 $(cd alembic && alembic upgrade head)
 python3 api/swagger-yaml-to-html.py < api/swagger.yml > api/index.html
-flask run
+serverless wsgi serve
```

Comment:
this is failing for me

```bash
(venv) david@david-Oryx-Pro:~/dev/personal/housinghub/backend$ serverless wsgi serve
Serverless: Using Python specified in "runtime": python3.6
Unable to import werkzeug (run: pip install werkzeug)
(venv) david@david-Oryx-Pro:~/dev/personal/housinghub/backend$ npx serverless wsgi serve
Serverless: Using Python specified in "runtime": python3.6
Unable to import werkzeug (run: pip install werkzeug)
```

Reply:
Weird, works for me. I added instructions for both ways in the README now